components/helpers/parent-by-attribute.js

```diff
@@ -0,0 +1,16 @@
+define('parent-by-attribute', [], () =>
+    function getParentByAttribute(el, attName, attValue) {
```

Comment:
I forgot to copy the JSdoc from the helpers.js. I have done so now, and included the TODO.